### DISCUSSION TITLE
AI_M_BOT v2.6.1

### DISCUSSION THREAD

**Discussion by JiaPai12138** [JiaPai12138/Crossfirer#19] [Ideas]
* 次要修复
* Minor fixes
---
* 恢复真正的多进程截图+推理
* Restored real multiprocessing for screenshot and prediction
---
* [如果提示缺少dll文件，请下载[lib.7z](https://github.com/JiaPai12138/Crossfirer/releases/download/ex_v2.5.6/lib.7z)并解压然后将lib文件夹移动至安装目录下
* If you got error like "dll is missing" or "dll not found", please download [lib.7z](https://github.com/JiaPai12138/Crossfirer/releases/download/ex_v2.5.6/lib.7z) and unzip it, then move the whole "lib" folder to the installation path (merge with the origin "lib" folder)
---
* **安装密码：名蒸蛋柯南之疗养院战队**
* **Installation password: 名蒸蛋柯南之疗养院战队**
---
* **Future releases will be at project [AI_M_BOT](https://github.com/JiaPai12138/AI-M-BOT)**
* **未来版本将发布在[AI_M_BOT](https://github.com/JiaPai12138/AI-M-BOT)项目中**

<hr /><em>This discussion was created from the release <a href='https://github.com/JiaPai12138/Crossfirer/releases/tag/ex_v2.6.1'>AI_M_BOT v2.6.1</a>.</em>

**Comment by xmt3061123** [comment]
打开   请低调使用.bat    以后，闪退怎么办

**Reply by JiaPai12138** [reply]
使用[此处](https://github.com/JiaPai12138/Crossfirer/blob/main/Crossfirer_v2.x/%E8%AF%B7%E4%BD%8E%E8%B0%83%E4%BD%BF%E7%94%A8.7z)解压的bat文件

**Comment by nuobaitutu** [comment]
可以运行，但是无法控制鼠标

**Comment by DsaHen** [comment]
哈哈哈，老哥我用了以后成自动不瞄了，我一瞄准自动帮我弹开XD

**Comment by Myrtlechief** [comment]
想请问一下，2.6.1应该是安装文件吧，但是打不开，双击之后没有任何反应也没有什么提示，有什么解决办法吗

**Reply by JiaPai12138** [reply]
没有 这项目我很久没更了 转另外地方去了

**Comment by watashi-sys** [comment]
The password is incorrect sir
